src/Skoruba.IdentityServer4.Admin.BusinessLogic/Repositories/ClientRepository.cs

```diff
@@ -55,21 +54,29 @@ public Task<Client> GetClientAsync(int clientId)
             return pagedList;
         }
 
-        public async Task<List<string>> GetScopesAsync(string scope)
+        public async Task<List<string>> GetScopesAsync(string scope, int limit = 0)
         {
-            var identityResources = await _dbContext.IdentityResources.Where(x => x.Name.Contains(scope)).Select(x => x.Name).ToListAsync();
-            var apiResources = await _dbContext.ApiResources.Where(x => x.Name.Contains(scope)).Select(x => x.Name).ToListAsync();
+            var identityResources = await _dbContext.IdentityResources
```

Comment:
I see now how these extension methods are used so maybe your design makes  more sense seeing this call site. Still feels weird to me but at the moment I don't have a better suggestion.